security-command-center/snippets/pom.xml

```diff
@@ -45,6 +45,18 @@
       <version>2.45.0</version>
     </dependency>
 
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-securitycentermanagement</artifactId>
+      <version>0.20.0</version>
+    </dependency>
```

Comment:
why need for fixed version dependency when using libraries-bom?

Reply:
SecurityCenterManagement class was not getting picked up (was getting the errors) hence I added this dependency explicitly with latest version